tutorials/preprocessing/plot_20_rejecting_bad_data.py

```diff
@@ -1,202 +1,208 @@
-#     to averaged files, noise-covariance matrices, forward solution
-#     files, and inverse operator decompositions.
-#
-# The actual removal happens using :func:`pick_types <mne.pick_types>` with
-# `exclude='bads'` option (see :ref:`picking_channels`).
+# Annotating bad spans of data
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# The tutorial :ref:`tut-events-vs-annotations` describes how
+# :class:`~mne.Annotations` can be read from embedded events in the raw
+# recording file, and :ref:`tut-annotate-raw` describes in detail how to
+# interactively annotate a :class:`~mne.io.Raw` data object. Here, we focus on
+# best practices for annotating *bad* data spans so that they will be excluded
+# from your analysis pipeline.
+#
+#
+# The ``reject_by_annotation`` parameter
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# In the interactive ``raw.plot()`` window, the annotation controls can be
+# opened by pressing :kbd:`a`. Here, new annotation labels can be created or
+# existing annotation labels can be selected for use.
+#
+# .. image:: ../../_static/annotation-controls-default.png
+#    :alt: screenshot of MNE-Python annotation controls window
+#
+# .. sidebar:: Annotating good spans
+#
+#     The default "BAD\_" prefix for new labels can be removed simply by
+#     pressing the backspace key four times before typing your custom
+#     annotation label.
+#
+# You can see that default annotation label is "BAD\_"; this can be edited
+# prior to pressing the "Add label" button to customize the label. The intent
+# is that users can annotate with as many or as few labels as makes sense for
+# their research needs, but that annotations marking spans that should be
+# excluded from the analysis pipeline should all begin with "BAD" or "bad"
+# (e.g., "bad_cough", "bad-eyes-closed", "bad door slamming", etc). When this
+# practice is followed, many processing steps in MNE-Python will automatically
+# exclude the "bad"-labelled spans of data; this behavior is controlled by a
+# parameter ``reject_by_annotation`` that can be found in many MNE-Python
+# functions or class constructors, including:
+#
+# - creation of epoched data from continuous data (:class:`mne.Epochs`)
+# - independent components analysis (:class:`mne.preprocessing.ICA`)
+# - functions for finding heartbeat and blink artifacts
+#   (:func:`~mne.preprocessing.find_ecg_events`,
+#   :func:`~mne.preprocessing.find_eog_events`)
+# - covariance computations (:func:`mne.compute_raw_covariance`)
+# - power spectral density computation (:meth:`mne.io.Raw.plot_psd`,
+#   :func:`mne.time_frequency.psd_welch`)
+#
+# For example, when creating epochs from continuous data, if
+# ``reject_by_annotation=True`` the :class:`~mne.Epochs` constructor will drop
+# any epoch that partially or fully overlaps with an annotated span that begins
+# with "bad".
+#
+#
+# Generating annotations programmatically
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
```

Comment:
Perhaps clarify that this is advanced usage and that what follows may be more practically handled by standard event detection.